cardano-cli/src/Cardano/CLI/Shelley/Run/Read.hs

```diff
@@ -100,6 +100,7 @@ data MetadataError
   | MetadataErrorValidationError !FilePath ![(Word64, TxMetadataRangeError)]
   | MetadataErrorDecodeError !FilePath !CBOR.DecoderError
   | MetadataErrorNotAvailableInEra AnyCardanoEra
+  deriving Show
```

Comment:
We appear to not use this `Show` instance anywhere.

Reply:
It is now necessary because `MetadataError` are embedded within `ShelleyGovernanceCmdError` which requires `Show`.

Reply:
Ah ok.  We may remove that instance of `Show` as well, but that doesn't have to be done in this PR.